web_widget_image_webcam/static/src/xml/web_widget_image_webcam.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<template>
+    <t t-extend="FieldBinaryImage">
+        <t t-jquery=".oe_form_field_image_controls" t-operation="after">
```

Comment:
You already have an icon bar in the image widget, so please just put your icon inside it. Remember to update the screenshot too.